### DISCUSSION TITLE
Transform/Organize Fields - give option to include only rather than exclude

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#54191] [Dashboards, Panels & Visualizations]
Greetings,

Sometimes we have no control over the data coming from some api or data that has tons of extraneous columns. Currently one has to configure grafana to use Transformation/Organize Fields to exclude. It is very tedious to exclude tons of columns.

![image](https://user-images.githubusercontent.com/900951/186528692-182a8616-90b1-417e-ad31-6a2c0ef08e62.png)

What is recommended here is to implement an **_include_** Organize Fields transformation. That way one selects only 3 fields to include out 100s, instead of clicking on 100s to exclude just for 3 of them to be included 😉 

Thanks!

**Comment by isaccavalcante** [comment]
I stumbled upon this today. I'm also interested in this feature. My use case is that I'm filtering kubernetes logs, and not all of them have the same labels. So when a new set of label appears I have to manually add the new labels to the ecluded list.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
